README.builder.md

```diff
@@ -1,38 +1,96 @@
+  4. At most 3 block bids are allowed to be sent at the same height from the same builder.
+
+Here are some sentry APIs that may interest a builder:
+
+  1. `mev_bestBidGasFee`. It will return the current most profitable reward that the validator received among all the blocks received from all builders. The reward is calculated as: `gasFee*(1 - commissionRate) - tipToBuilder`. A builder may compare the `bestBidGasFee` with a local one and then decide to send the block bid or not.
+  2. `mev_params`. It will return the `BidSimulationLeftOver`,`ValidatorCommission`, `GasCeil` and `BidFeeCeil` settings on the validator. If the current time is after `(except block time - BidSimulationLeftOver)`, then there is no need to send block bids anymore; `ValidatorCommission` and `BidFeeCeil` helps the builder to build its fee charge strategy. The `GasCeil` helps a builder know when to stop adding more transactions.
+
+Builders have the freedom to define various aspects like pricing models for users, creating intuitive APIs, and define the bundle verification rules.
+
+### Setup with Example Builder
+
+Step 1: Find Validator Information
+For validators that open MEV integration, the public information is shown at [bsc-mev-info](https://github.com/bnb-chain/bsc-mev-info). Builders can also provide information here to the validator.
+
+Step 2: Set up Builder.
+The builder must sign the bid using an account, such as the etherbase account specified in the config.toml file.
+
+```toml
 [Eth.Miner.Mev]
-Enabled = false
-ValidatorCommission = 100
-BidSimulationLeftOver = 50
-BuilderEnabled = true
-BuilderAccount = {{BUILDER_ADDRESS}}
+BuilderEnabled = true # open bid sending
+BuilderAccount = "0x..." # builder address which signs bid, usually it is the same as etherbase address
+
+# Configure the validator node list, including the address of the validator and the public URL. The public URL refers to the sentry service.
+[[Eth.Miner.Mev.Validators]]
+Address = "0x23707D3D71E31e4Cb5B4A9816DfBDCA6455B52B3"
+URL = "https://bsc-fuji.io"
 
 [[Eth.Miner.Mev.Validators]]
-Address = {{VALIDATOR_ADDRESS}}
-URL = {{VALIDATOR_URL}}
-...
+Address = "0x..."
+URL = "https://bsc-mathwallet.io"
 ```
 
-- `Enabled`: Whether to enable validator mev.
-- `BuilderEnabled`: Whether to enable the builder mev.
-- `BuilderAccount`: The account address to unlock of the builder.
-- `BidSimulationLeftOver`: The left over of the bid simulation.
-- `Validators`: A list of validators to bid for.
-  - `Address`: The address of the validator.
-  - `URL`: The URL of the validator.
+## FAQ
+
+1. Do builders fetch the in-turn proposer’s GasCeil to build block？
+    Yes, you could using RPC mev_params to query validator’s MEV information before building block, it can help to 1) calculate a valid header with gas no more than GasCeil; 2) calculate the left bidding time by BidSimulationLeftOver; 3) calculate suitable builderFee by validatorCommission.
```

Comment:
can you merge the contend of `https://github.com/bnb-chain/bsc-builder/blob/master/README.builder.md`, and delete that file then?